### DISCUSSION TITLE
flutter 端什么时候上线捏

### DISCUSSION THREAD

**Discussion by obweix** [Tencent/tdesign#30] [Q&A]
flutter 端什么时候上线捏

**Comment by pattybaby110** [comment]
flutter还处在很初期的阶段...

**Reply by XuJin186** [reply]
这也太初期了一点   我想flutter.最终会被官方放弃  因为大部分逻辑无法复用  只能用dart重写  也许rn是个不错的选择

**Reply by Transformi** [reply]
官网显示 移动端的react 是指rn吗？
